Database/Corrections/sodQuestFixes.lua

```diff
@@ -3164,10 +3164,70 @@ function SeasonOfDiscovery:LoadQuests()
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 10,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.ROGUE,
+            [questKeys.objectivesText] = {"Loot the Lighthouse Stash under the right of the tree to receive the rune."},
+            [questKeys.requiredSpell] = -424984,
+            [questKeys.zoneOrSort] = sortKeys.ROGUE,
+        },
+        [90210] = {
+            [questKeys.name] = "Saber Slash",
+            [questKeys.startedBy] = {nil,{407731}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 10,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.ROGUE,
+            [questKeys.objectivesText] = {"Loot the Stonemason's Toolbox on the platform behind the quest givers at Stonewrough Dam to receive the rune."},
+            [questKeys.requiredSpell] = -424984,
+            [questKeys.zoneOrSort] = sortKeys.ROGUE,
+        },
+        [90211] = {
+            [questKeys.name] = "Saber Slash",
+            [questKeys.startedBy] = {{210549}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 15,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.ROGUE,
+            [questKeys.objectivesText] = {"Approach the Defias Scout in Stealth and Pickpocket them to receive the rune."},
+            [questKeys.requiredSpell] = -424984,
+            [questKeys.zoneOrSort] = sortKeys.ROGUE,
+        },
+        [90212] = {
+            [questKeys.name] = "Saber Slash",
+            [questKeys.startedBy] = {nil,{407457}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 10,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.ROGUE,
+            [questKeys.objectivesText] = {"Take a left from the main gate, go up and around to the right and jump down onto the wall behind the stable and drop on to the roof and open the Stable Hand's Trunk to receive the rune. NOTE: You need 80 Lockpicking to open the trunk."},
+            [questKeys.requiredSpell] = -424984,
+            [questKeys.zoneOrSort] = sortKeys.ROGUE,
+        },
+        [90213] = {
+            [questKeys.name] = "Saber Slash",
+            [questKeys.startedBy] = {nil,{409131}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 10,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.ROGUE,
+            [questKeys.objectivesText] = {"Use Sprint and jump to the Rusty Chest on the platform and open it to receive the rune. NOTE: You need to have learnt Sprint to make the jump."},
```

Comment:
Instead of using Sprint bringing a [Swiftness Potion](https://www.wowhead.com/classic/item=2459/swiftness-potion) brings the same result.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5609/commits/82f0883c19e082887899e9a1192485228c484edf